src/validation/category/delete.ts

```diff
@@ -0,0 +1,7 @@
+import * as yup from 'yup'
+
+const deleteCategoryValidation = yup.object().shape({
+  categoryId: yup.number().required()
```

Comment:
I think this needs to be positive and this might be the way to validate the positive number
```suggestion
  categoryId: yup.number().positive().required()
```